docs/UserGuide.md

```diff
@@ -1,172 +1,191 @@
+        - 5.1.4 [Update a local course: `localcourse update`](#514-update-a-local-course-localcourse-update)
+        - 5.1.5 [Search for a local course by an attribute: `localcourse search`](#515-search-for-a-local-course-by-an-attribute-localcourse-search)
+        - 5.1.6 [Sort all local courses by an attribute: `localcourse sort`](#516-sort-all-local-courses-by-an-attribute-localcourse-sort)
+    - 5.2 [Commands for partner courses](#52-commands-for-partner-courses)
+        - 5.2.1 [List all partner courses: `partnercourse list`](#521-list-all-partner-courses-partnercourse-list)
+        - 5.2.2 [Add a partner course: `partnercourse add`](#522-add-a-partner-course-partnercourse-add)
+        - 5.2.3 [Delete a partner course: `partnercourse delete`](#523-delete-a-partner-course-partnercourse-delete)
+        - 5.2.4 [Update a partner course: `partnercourse update`](#524-update-a-partner-course-partnercourse-update)
+        - 5.2.5 [Search for a partner course by an attribute: `partnercourse search`](#525-search-for-a-partner-course-by-an-attribute-partnercourse-search)
+      - 5.2.6 [Sort all partner courses by an attribute: `partnercourse sort`](#526-sort-all-partner-courses-by-an-attribute-partnercourse-sort)
+    - 5.3 [Commands for universities](#53-commands-for-universities)
+        - 5.3.1 [List all universities: `university list`](#531-list-all-universities-university-list)
+        - 5.3.2 [Search for a university by an attribute: `university search`](#532-search-for-a-university-by-an-attribute-university-search)
+      - 5.3.3 [Sort all universities by an attribute: `university sort`](#533-sort-all-universities-by-an-attribute-university-sort)
+    - 5.4 [Commands for mappings](#54-commands-for-mappings)
+        - 5.4.1 [List all mappings: `mapping list`](#541-list-all-mappings-mapping-list)
+        - 5.4.2 [Add a mapping: `mapping add`](#542-add-a-mapping-mapping-add)
+        - 5.4.3 [Delete a mapping: `mapping delete`](#543-delete-a-mapping-mapping-delete)
+        - 5.4.4 [Search for a mapping by an attribute: `mapping search`](#544-search-a-mapping-by-attributes-mapping-search)
+        - 5.4.5 [Sort all mappings by an attribute: `mapping sort`](#545-sort-a-mapping-by-attributes-mapping-sort)
+    - 5.5 [Commands for notes](#55-commands-for-notes)
+        - 5.5.1 [List all notes: `note list`](#551-list-all-notes-note-list)
+        - 5.5.2 [Add a note: `note add`](#552-add-a-note-note-add)
+        - 5.5.3 [Delete a note: `note delete`](#553-delete-a-note-note-delete)
+        - 5.5.4 [Update a note: `note update`](#554-update-a-note-note-update)
+        - 5.5.5 [Tag a note: `note tag`](#555-tag-a-note-note-tag)
+        - 5.5.6 [Clear all tags from a note: `note cleartag`](#556-clear-all-tags-in-a-note-note-cleartag)
+    - 5.6 [View help : `help`](#56-view-help--help)
+    - 5.7 [Exit SEPlendid: `exit`](#57-exit-seplendid--exit)
+    - 5.8 [Save the data](#58-save-the-data)
```

Comment:
Links will not work if you do this. 

As discussed, this part of the UG is assigned to me, and I have spent a long time fixing it. Please do not replace the table of contents with your own, as we are tasked to reflect contribution appropriately.

Reply:
I understand that this part is your part. This pull request was still a draft at the point of your review. The non-functioning parts of the UG is not meant to be merged. Will not be making changes to your assigned part unless it does not work. Thank you

Reply:
Thanks for the clarification, let me know if there is anything amiss for the TOC. Feel free to make changes if they are minor, but I'll be checking all headers and links prior to submission. 👍